optimizely/java-sdk
# feat: add "json" type to FeatureVariable

## Summary
- Add "json" type to FeatureVariable
- Add "string" + "json" subtype for backward compatibility

## Test plan
- Verify that subtype is converted to json type internally
- Verify that the new data type does not crash SDK
